common/config.js

```diff
@@ -33,13 +43,48 @@ const configDefaults = {
                 default: return `https://broad-bond-${dsdeEnv}.appspot.com`;
             }
         })(),
-    dataObjectResolutionHost:
+    bioDataCatalystHost:
+        (() => {
+            switch (marthaEnv) {
+                case ENV_MOCK:
+                    return HOST_MOCK_DRS;
```

Comment:
ToL:

"someone" didn't test their code logic in a previous PR. (Git blame is liar, don't trust it.)

Perhaps this entire bit of logic could be better tested if the export were actually a function that takes in an `env: Object` and returns an `Object`, kind of like `supertest`'s [export](https://github.com/visionmedia/supertest/blob/v6.1.3/index.js#L18).

Then all of the various conditions could be verified via unit tests _before_ they reach those various actual `env` situations, especially production.

I think that is a big amount of testing to pile onto this PR for a previous sin, so marking it ToL and perhaps a future ticket.